hips/hip-0012.md

```diff
@@ -45,7 +45,8 @@ This document is intended to address these criticisms by presenting a clear plan
 It describes the process in detail, providing both a roadmap for us to follow and a point of reference for community members to use when contributing changes.
 
 The process is based on the following ideas:
-- Requiring changes to be documented, either as "Helm 4 proposals" (H4HIP) or standard HIPs. Allowing both Helm community members and maintainers to review them.
+- Requiring changes to be documented, either as reduced "Helm 4 proposals" (H4HIP) or standard HIPs.
+Allowing both Helm community members and maintainers to review them.
```

Comment:
This is just whitespace (I've put every sentence on a newline in this file, as markdown formats paragraphs regardless. And it makes it easier to modify for me)